Development/boost/thread/win32_condition_variable.hpp

```diff
@@ -0,0 +1,444 @@
+// based on the <boost/thread/win32/condition_variable.hpp>
```

Comment:
Add the file's version of Boost in this comment

Reply:
pushed in